Reference Arch: AWS: Be more explicit about regional outages (#1674)
hashicorp/terraform-website: content/source/docs/enterprise/before-installing/reference-architecture/aws.md

> An identical infrastructure should be provisioned in a secondary AWS
> Region. Depending on recovery time objectives and tolerances for

Reviewer: ```suggestion
We recommend provisioning an identical infrastructure in a secondary AWS
Region. Depending on recovery time objectives and tolerances for
```

additional cost to support AWS Region failure, the infrastructure can be

> running (Warm Standby) or stopped (Cold Standby). Please note that with _Standalone_ implementation mode, only one Terraform Enterprise instance can be running against the same database. This deployment acts to minimize the Mean Time To Recovery (MTTR) in the event of a regional failure, avoiding the need to replicate and stand up the data plane infrastructure during an outage. In the event of the primary AWS Region hosting the Terraform Enterprise application fail, the secondary
> AWS Region will require some configuration before traffic is directed to
> it along with some global services such as DNS.

Reviewer: ```suggestion
running (Warm Standby) or stopped (Cold Standby). Please note that with _Standalone_ implementation mode, only one Terraform Enterprise instance can be running against the same database. 

This deployment acts to minimize the Mean Time To Recovery (MTTR) in the event of a regional failure, avoiding the need to replicate and stand up the data plane infrastructure during an outage. If the primary AWS Region hosting the Terraform Enterprise application fails, you will need to perform some configuration before traffic is directed to the secondary AWS Region:
```

AWS Region will require some configuration before traffic is directed to
it along with some global services such as DNS.

> Please note that with _Standalone_ implementation mode, only one Terraform Enterprise instance can be running against the same database. This deployment acts to minimize the Mean Time To Recovery (MTTR) in the event of a regional failure, avoiding the need to replicate and stand up the data plane infrastructure during an outage.

Reviewer: I think it might be helpful to move this up so that it comes right after the warm and cold standby sentence. Right now, it interrupts the flow because it comes in between the sentence about configuration and the following examples.
